### PR TITLE
Antonio2

### PR DESCRIPTION
**Pull request by Aktrax**
It has a couple easter eggs. If too much tell me and I will remove them.

### CODE REVIEW

File: step-2/application/src/components/App/App.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.content {
+	height: 95%;
```
Comment (LesleyMerks) [inline step-2/application/src/components/App/App.css]: this formatting in quite of. You can install tools for this to fix this formatting automatically: 
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: step-2/application/src/components/App/App.tsx [hunk 2]
```diff
@@ -58,17 +62,49 @@ export interface GitHubData {
 interface IAppProps {
   fetchData: () => void;
   commits: GithubCommit[];
+  error: string | null;
+  isLoading: boolean;
 }
 
 export class App extends React.Component<IAppProps, AppState> {
+  loadingTexts: string[] = [
```
Comment (LesleyMerks) [inline step-2/application/src/components/App/App.tsx]: why is the in the context of the entire class? only getLoadingText is using this, correct?

File: step-2/application/src/modules/Commits/Actions.ts [hunk 3]
```diff
@@ -1,47 +1,49 @@
...
+          return p;
+        }
+        return data.json();
+      })
+      .then(
+        (data: GitHubData[]) => {
+          dispatch(fetchDataResult(Status.success, data, ''));
+        },
+        error => {
+          dispatch(fetchDataResult(Status.failure, [], 'Alerting is not supported in this alert.'));
```
Comment (LesleyMerks) [inline step-2/application/src/modules/Commits/Actions.ts]: you can/should dispatch an error action here

File: step-2/application/src/components/App/App.tsx (hunk 2 above)
Comment (Aktrax) [reply step-2/application/src/components/App/App.tsx]: yes. I could move it to the context of the method

File: step-2/application/src/modules/Commits/Actions.ts (hunk 3 above)
Comment (Aktrax) [reply step-2/application/src/modules/Commits/Actions.ts]: What is an error action?

**Pull request closed without merging**